Tasks/XcodeV5/xcode.ts

```diff
@@ -5,6 +5,11 @@ import utils = require('./xcodeutils');
 
 import { ToolRunner } from 'azure-pipelines-task-lib/toolrunner';
 
+const defaultSimulators = {
```

Comment:
Maybe it makes sense to use an enum instead of the object in this case?

Reply:
Despite the fact that after compiling a ts string enum a get an object it seems to me that the enum is rather a plain list of strings, but a key/value dictionary. Actually, it is a lookup table that is used to match the platform and simulator, which are different things. I've added types to the object.

Reply:
Updated simulators constants